### DISCUSSION TITLE
SyncVar Delta Compression

### DISCUSSION THREAD

**Discussion by Canglangzi** [FirstGearGames/FishNet#730] [Feature Request]
Games such as cs2 and valorant all use this synchronization method. 
This synchronization method is best able to deal with packet loss problems.

I hope to leave a similar channel where I can see the feedback of packet loss and debug it and resend the packet.

Unreliable sending, even if the packet is lost, I will resend the next frame

And it is very likely that a large number of api breaking changes will be required

Synchronous rpc can be optimized later. In my test, one rpc was 11 bytes empty.

Does the bottom layer still need some incremental compression broadband optimization, etc.

**Comment by FirstGearGames** [comment]
Deltas are planned across the board and already in development.

PS: an empty RPC from the server should be 2 bytes for reliable, 3 for unreliable. From the client 4 reliable, 5 unreliable.
